storm-buildtools/storm-maven-plugins/src/main/java/org/apache/storm/maven/plugin/versioninfo/VersionInfoMojo.java

```diff
@@ -113,6 +113,7 @@ private String getBuildTime() {
      * @return SCM in use for this build
      * @throws Exception if any error occurs attempting to determine SCM
      */
+    @SuppressWarnings("checkstyle:AbbreviationAsWordInName")
```

Comment:
I think for stuff in this module, feel free to just change the names. This isn't part of a user-facing API.

Reply:
I'm starting to not like this rule :)

Reply:
It's fine for new code I think. I agree that it's annoying for our existing code.